webmock.gemspec

```diff
@@ -16,10 +16,8 @@ Gem::Specification.new do |s|
   s.rubyforge_project = 'webmock'
 
   patron_version      = (RUBY_VERSION <= '1.8.7') ? '0.4.18'   : '>= 0.4.18'
-  addressable_version = (RUBY_VERSION) > '1.8.7'  ? '>= 2.3.6' : '< 2.4.0'
```

Comment:
I don't know if you want to remove the addressable limit completely,
before the recent change it was `>= 2.3.6` right?